doc/user/content/ingest-data/_index.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "Ingesting data"
+description: "How to ingest data into Materialize from external systems."
+disable_list: true
+menu:
+  main:
+    identifier: "ingest-data"
+    name: "Ingest data"
+    weight: 11
+---
+
+Materialize can ingest data from various external systems:
+
+{{< multilinkbox >}}
+{{< linkbox title="Databases (CDC)" >}}
+- [PostgreSQL](/ingest-data/postgres/)
+- [MySQL](/ingest-data/mysql/)
+- [SQL Server](/ingest-data/cdc-sql-server/)
+- [CockroachDB](/ingest-data/cdc-cockroachdb/)
+{{</ linkbox >}}
+{{< linkbox title="Message Brokers" >}}
+- [Kafka](/ingest-data/kafka/)
+- [Redpanda](/sql/create-source/kafka)
+- [Other message brokers](/integrations/#message-brokers)
+{{</ linkbox >}}
+{{< linkbox title="Webhooks" >}}
+- [Amazon EventBridge](/ingest-data/webhooks/amazon-eventbridge/)
+- [Segment](/ingest-data/webhooks/segment/)
+- [Other webhooks](/sql/create-source/webhook)
+{{</ linkbox >}}
+{{</ multilinkbox >}}
+
+## Sources and clusters
+
+Materialize ingests data from external systems using
+[sources](/concepts/sources/). For the sources, you need to associate a
+[cluster](/concepts/clusters/) to provide the compute resources needed to ingest
+data.
+
+{{% tip %}}
+
+- If possible, dedicate a cluster just for sources.
+
+- When you create a new source, Materialize performs a one-time [snapshotting
+  operation](#snapshotting) to initially populate the source in Materialize.
+  Snapshotting is a resource-intensive operation that can require a significant
+  amount of CPU and memory. Consider using a larger cluster size during
+  snapshotting. Once the snapshotting operation is complete, you can downsize
```

Comment:
For future verifications: 

- It may not be the case for certain sources.
- Maybe after a certain size, it might not buy as much improvement.
- For  upsert sources, emphasize this a bit more + dedicated cluster for upsert sources (as in separate these sources from other sources)

Reply:
> amount of CPU and memory.

This is really only true for upsert sources. OLTP sources do not have elevated memory usage. A larger cluster can make it go faster but its not strictly necessary. 

The dedicated cluster for sources is a good idea! What if we have that as it's own tip and then move the rest to something more upsert specific?

Reply:
👍 Will add a patch for the upsert-specific stuff.

Reply:
Added  patch https://github.com/MaterializeInc/materialize/pull/31468/commits/2acdb6bc2dc5064ea9b1de89329d371aa3a49a64